File: CC2_ComProg1/CC2_Finals/Quizzes/PasswordGenerator.java
/**
 * @author pao
 * BSCS 1B
 */
import java.util.Scanner;
public class PasswordGenerator {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        StringBuilder password = new StringBuilder();
        System.out.println("----------------------");
        System.out.println("| Password Generator |");
        System.out.println("----------------------");
        System.out.print("Enter your first name     : ");
        String firstName = sc.nextLine();
        System.out.print("Enter your last name      : ");
        String lastName = sc.nextLine();
        System.out.print("Enter your birthyear      : ");
        String year = sc.nextLine();
        System.out.print("Enter your favortie color : ");
        String color = sc.nextLine();
        password.append(lastName.substring(0, 3));
        password.append(year.substring(2));
        password.append(firstName.substring(0, 4));
        password.append(color.charAt(0));
        
        System.out.println("Generated Password: " + password.toString().toLowerCase());
    }
}

File: Practice Codes/CC2 Java to Python/Finals/Quizzes/password_generator.py
print("|------------------------------|")
print("|      Password Generator      |")
print("|------------------------------|")
firstName = input("Enter your first name     : ")
lastName = input("Enter your last name      : ")
year = input("Enter your birth year     : ")
color = input("Enter your favorite color : ")
password = lastName[:3] + year[2:] + firstName[:4] + color[0]
print("Generated Password: " + password.lower())